auth/migrations/0002_subscriptions.sql

```diff
@@ -0,0 +1,26 @@
+CREATE TABLE IF NOT EXISTS subscriptions (
+  subscription_id TEXT PRIMARY KEY,
+  account_name TEXT NOT NULL,
+  type TEXT NOT NULL,
+  quantity INT DEFAULT 1,
+  updated_at TIMESTAMPTZ DEFAULT CURRENT_TIMESTAMP, 
+  UNIQUE (account_name, type),
+  FOREIGN KEY (account_name) REFERENCES users(account_name)
+);
+
+-- Create a trigger to automatically update updated_at
+CREATE TRIGGER sync_users_updated_at
+BEFORE UPDATE
+ON subscriptions
+FOR EACH ROW
+EXECUTE PROCEDURE sync_updated_at();
+
+-- Insert existing subscriptions into the new subscriptions table, all of which are of the pro type
+INSERT INTO subscriptions (subscription_id, account_name, type)
+SELECT subscription_id, account_name, 'pro'
+FROM users
+WHERE subscription_id IS NOT NULL;
+
+-- Drop the subscription_id column from the users table
+ALTER TABLE users
+DROP COLUMN subscription_id;
```

Comment:
Hmm, should we drop the `tier` too?

Reply:
So the way I've thought about this is keeping the subscriptions separate, that is keeping the pro subscription as is, and creating more general logic for addons. But it would be a lot nicer if we could make the logic general already... The pro subscription has some additional logic that I didn't consider necessary for RDS, but perhaps it isn't necessary for the pro tier either, perhaps we can just set the pro tier limits on the claim based on the subscription like we do for RDS.

Reply:
Ok, I can't currently remove this because that will screw with our admin and deployer tier accounts... something for another day.